src/QirTools/pyqir/qirlib/runtime.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.18.2106148911-alpha" />
```

Comment:
Is there a way to not hardcode the package version numbers here? If not, we should make sure to exempt this from the version number update steps in bootstrap.ps1.